src/components/toast/toast.component.tsx

```diff
@@ -110,8 +110,11 @@ export const Toast = React.forwardRef<HTMLDivElement, ToastProps>(
     useEffect(() => {
       if (!disableAutoFocus) {
         if (open) {
-          focusedElementBeforeOpening.current = document.activeElement as HTMLElement | null;
-          toastContentNodeRef.current?.focus();
+          // setTimeout needed as otherwise this runs before the ref is populated
+          setTimeout(() => {
+            focusedElementBeforeOpening.current = document.activeElement as HTMLElement | null;
+            toastContentNodeRef.current?.focus();
+          }, 0);
```

Comment:
thanks - I think you're right. In fact I just checked MDN and can confirm you are right.

But precisely because this isn't obvious to everyone (in that, it at least isn't obvious to me!), I prefer to explicitly state that the delay is 0, so people reading the code don't have to wonder "what does setTimeout do when you leave out the delay argument?".

I've just done a quick search of the project and, as I suspected, there are other uses of `setTimeout` with an explicit delay of 0 - although I'm sure at least some of those were added by me 😂 But I also found some others which omit the second argument entirely, as you're suggesting. So consistency isn't an argument either way 😂 

I still prefer it with the explicit 0, but I'm not going to argue to forcefully against it if you insist 😄

Reply:
I love a bit of `consistently inconsistent` 🤣 

I'm more than happy for you to leave it as is though. Might be something we should put to the team and come up with a consistent approach for though 👍🏻